fedoragsearch-transforms/islandora_transforms/slurp_all_MODS_to_solr.xslt

```diff
@@ -429,6 +426,17 @@
         </xsl:attribute>
         <xsl:value-of select="$value"/>
       </field>
+    </xsl:if>
```

Comment:
@cdeaneGit hey, I'm confused as to where/why this change came in. Am I missing something somewhere? Any suggestions or reminders would be most helpful. Thanks!

Edit: referring to lines 429-439

Reply:
The way I resolved the conflicts was to pull all the code that was in institutional_repository/slurp_all_MODS_to_solr.xslt into TRAC-891/slurp_all_MODS_to_solr.xslt.  The comparison between the two branches kept having changes that were outside of the template I was working on.  So where this particular change came from, I do not know.